app_allocator/tests/test_judge.py

```diff
@@ -9,3 +15,53 @@ def test_judge_mark_as_done_creates_event(self):
         judge.mark_as_done()
         assert Event.ticks == prior_ticks + 1
         assert Event.all_events[-1].fields['subject'] == judge
+
+    def test_judge_mark_as_done_zeroes_judge_remaining_count(self):
+        judge = Judge()
+        judge.remaining = 10
+        judge.mark_as_done()
+        assert judge.remaining == 0
+
+    def test_complete_startups_creates_events_for_all_startups(self):
+        judge = Judge()
+        startups = [Startup() for _ in range(10)]
+        judge.startups = startups[:]
```

Comment:
Not a blocker, but I don't like this idiom and I'm not alone according to https://stackoverflow.com/questions/2612802/how-to-clone-or-copy-a-list.  Personally, I think `judge.startups = list(startups)` is the most reasonable, `copy.copy(list)` seems bit over the top.

Reply:
It's pretty much the standard way to do it - some random dude's crankiness on some random website notwithstanding - which is why I used it. But if we don't mind being unidiomatic, I'm a big supporter of well-motivated idiosyncracy over blind adherence to convention.